cluster/saltbase/salt/kubelet/default

```diff
@@ -190,5 +188,14 @@
 
 {% set kubelet_auth = "--anonymous-auth=false --authorization-mode=Webhook --client-ca-file=" + pillar.get('ca_cert_bundle_path', '/var/lib/kubelet/ca.crt') %}
 
+# TODO(mtaufen): Convert the old switch for standalone mode to the new switch.
```

Comment:
Since you are ripping out the plumbing everywhere else I don't see a reason not to do it at the same time for the salt configs. If we get this PR in right after the code slush lifts then it'll have plenty of time to bake before 1.7.